YACS-RCOS/yacs
# DevOps Overhaul

This has been a long time coming. This PR aims to move to a new, sleeker ops flow. The goal is to move the compose config and ops scripts out of here and into a master repo, allowing for more straightforward deployment and development. see https://github.com/YACS-RCOS/yacs-orchestra for more info and all the stuff that this moves out of here.

 In summary:
- Move nginx out of here and into its own repo
- Reconfigure travis to test the pre-dockerized app (for now)
- Move certs to a volume
- Remove docker-compose config (see above)
- Remove custom scripts (see above)

This also merges in some hotfixes that were made in production:
- increase nginx max header size
- fix logging and caching in production
- update for spring 2018
- probably something else I forgot

:)
